LanguageFeatures/Private-fields-promotion/not_promotable_A04_t04.dart

```diff
@@ -37,5 +37,12 @@ abstract class C<T> {
 }
```

Comment:
Github won't allow me to comment on line 15-16, but we should note that it's about an abstract getter, not just about an abstract class (where the getter could be declared, and it could still be concrete).

By the way, the abstract getter could also be implicitly induced by an abstract instance variable declaration, haven't checked whether that's covered.

Reply:
Changed the descriprion to mention an abstract getter and updated `not_promotable_A05_t04.dart` to test an abstract field (I remember that spec doesn't use word `field` and it's better to use `instance variable` but if the feature is called `Private fields promotion` I suppose it's Ok to use word `field`)